docs/rules/no-unstable-nested-components.md

```diff
@@ -124,7 +124,7 @@ function Component() {
   "off" | "warn" | "error",
   {
     "allowAsProps": true | false,
-    "customValidators": [] /* optional array of validators used for propTypes validation */
```

Comment:
why is this being removed here?

Reply:
Removed the cleanup and added some context to https://github.com/jsx-eslint/eslint-plugin-react/issues/3664

Reply:
oof, i see. in that case we definitely should have a separate PR to either remove it from the schema (a breaking change, unfortunately, even though it doesn't do anything) or to add the functionality (which could be a patch)

Reply:
@ljharb While we figure out what to do with the other option, what do you think about progressing this feature?